### DISCUSSION TITLE
Welcome to equiformer-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/equiformer-pytorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abdalazizrashid** [comment]
Any ideas how to use this model for conditional generation of crystals?

**Reply by lucidrains** [reply]
i would go the score matching / ddpm route https://arxiv.org/abs/2205.15019

**Reply by lucidrains** [reply]
start with invariance, as it is a lot cheaper than equivariance https://github.com/lucidrains/invariant-point-attention

**Comment by zhuyu-cs** [comment]
Hi, is it possiable for predicting the adjacent matrix？Since in molecule-building，the connection information is still important.

**Reply by lucidrains** [reply]
@zhuyu-cs i'd be doing your research for you, but the route i would take is to extract all the type 1 attention maps, concat them, and do a small MLP head for predicting adjacency

**Comment by radiradev** [comment]
How does this transformer compare with en_transformer in terms of performance and use cases?
